fixtures/example-app/haul.config.js

```diff
@@ -0,0 +1,7 @@
+import { createWebpackConfig } from 'haul';
```

Comment:
Maybe will be nice to delete this file in one of the fixtures to ensure our default config works properly

Reply:
Agree.